### DISCUSSION TITLE
Preview for video

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-curator#100] [Ideas]
Hello, i think it would be nice, if plugin can support video mime type too
Maybe add default html5 video tag
right now it's looks like this
![image](https://user-images.githubusercontent.com/1766890/217014169-6d60bc2b-205a-44cf-a59f-b009c6bc7f87.png)

**Comment by awcodes** [comment]
So you want a playable video?

**Reply by lotarbo** [reply]
yes, similar to field fileupload in filament

**Comment by awcodes** [comment]
Added in v2.1.0.
